### PR TITLE
BAU: BJGill's PR for enabling coverage & codacy

### PR DESCRIPTION
**Pull request by tomrosier**
Branch moved from fork to our repo

### CODE REVIEW

**Comment by CharlesIC** [comment]
Why the rename of `GenericExceptionMapper` to `CatchAllExceptionMapper`?
Noting this is unrelated to enabling coverage

File: .travis.yml [hunk 1]
```diff
@@ -3,10 +3,16 @@ language: java
...
+    - secure: "E/Ui52SP8jC9kvP9RxYcvrP2Q3BPyKdi/almVWEfWetBKDH7g9M1bl6pgTLTrI31CZGS0O2MtOC05vTp7KkksveBwhbvXLOcb934bLpeOP0son3gXDH5ANTwOq98amFlyKbqzZre0BUMeIPtHc1ZoGc5s6q5a+nZ78zgyjiDVc9StlRx38QcF0rM7S9zzcIqu0eNoLTI4pDI/cP0STpIG68rsGKQSVUI5dFAGC0ZdVIEH7rSP/G5rPbX4scvwiQ2iBaPuaOkPvgfAW7Th4reQK3/97C+ShfC3aNfPfCYZoSYBROHSrgFqvSsEOGjiKnuqaz07RcGsh1tAXLrzcYVey3YExxiPVIYEY8q5P5nCjE0pT65AJqnnXT0gJ/+zLV9I5x4DPEBRcgURDAhVgMRTlOobE0VLb2rvsTdUZpiopVoAQUQEH1s8rD1LA1EqethRI3IHBv9jK63ca0gwwQcKh6KxZE8YxjsiVTXPjRu64XFz0OlH4PBn7ZNja/b0aGeaQt/qoxGHTQ3/xkuiVArg1WfVKo1ZW9jOG5R2vOiV6Lbxkmcb+pkMUF9X8JvzUYNUjNFqHA2119lXbamVKG2gAzi6dlmFldpyyU8rdJONdrRmdAlAKq10Cre2QEPFzh1ehHIMrRVlxT0pi7BQM7aH2sKZzFeifhpL5+0J698emo="
```
Comment (bjgill) [inline .travis.yml]:
```suggestion
    - secure: "RWxtPrNRQkNZB4pW6FTXti9WdNntvaevSvk6V/E37XLoXJlqn7GxPcNSU2WBKY0kC7nq0QEkR44vPJwQ5zJvr5LFpwRjI5zw6WVeFV8N2n7zNDxIAD4zfUzJOzWmEK6ZJhtL5ISqjf/9L++VcbOys4pJX/2IIeqIVlivNCaOou5cj1owPaG80otzfcpH+aX2voSxpAtc2pyzWfN30O64NCJIdmo1fuaoQuNK2lqp+5NQhCqWDgWVnrGmw06Brwcw4XodvMGzNqbASq+o2mc8+vFQHI9wLScrRA4X3JfvHho5icUQoeKN7/ohRzBs+BoU/uURUfKfIam6cObJbFlcn9UlXS9l9LbpRaiUxvnRaXoas5OE3mSnCD+UfgzFf7cFYBSLcsQNOO5pzxvGGc8NzMN3XVo13vDAf6SOCfiR4cjKx9YdXvK10aHOA72zY9giFcmWieocT8Yth+EbXi7jshSuYKEB5ig8tqH4Os4vXVKNGqow4vxZf4f9AxeAK8mBAlD5Mbd00JU2euzkOF3DSgzEb1uxg4BzM7YBrm74b7F3X2GO0mDsKsP8ZsLy0p+cLORXSrcPt3wQTkDMo7jGaVkalxKCmTUU48q4v/5VLGMMVTbU861/5flC/CLGU19ABFnCG46AorcDagVcsTA9EOqllHVi2KwJPVxLr+CFx88="
```

Updated GITHUB_PAT signed for travis-ci.com

**Comment by CharlesIC** [comment]
@bjgill Can you please double check the secure variable?
```
The previous command failed, possibly due to a malformed secure environment variable.
Please be sure to escape special characters such as ' ' and '$'.
For more information, see https://docs.travis-ci.com/user/encryption-keys.
```

**Comment by bjgill** [comment]
I know what the problem is. I think I generated the encrypted variables for travis-ci.**org**, which is what the rest of Verify uses. You're using the newer travis-ci.**com**.

I've submitted suggestions re-encrypted correctly.

**Comment by CharlesIC** [comment]
Thanks, @bjgill 
Travis is running again now 🤞

File: .travis.yml [hunk 2]
```diff
@@ -20,6 +26,9 @@ before_cache:
 
 after_success:
   - snyk/monitor.sh
+  - ./gradlew jacocoTestReport
```
Comment (bjgill) [inline .travis.yml]:
```suggestion
  - ./gradlew jacocoRootReport
```

Make sure report aggregator gets run by Travis

**Comment by CharlesIC** [comment]
All green.... Let's merge!

**Comment by bjgill** [comment]
Thanks all for the help delivering this!

**Pull request merged**